src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,51 @@
-export const Pagination = () => {};
+type Props = {
```

Comment:
The provided props do not match the task description.
![image](https://github.com/mate-academy/react_pagination/assets/96409563/61e39ade-cb31-44d4-930d-d514ac1c76b1)